sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -518,7 +521,8 @@ class HealthcareRelation(DictMixin):
 
     :ivar relation_type: The type of relation, i.e. the relationship between "100mg" and
         "ibuprofen" in the document "The subject took 100 mg of ibuprofen" is "DosageOfMedication".
-    :vartype relation_type: str or ~azure.ai.textanalytics.HealthcareEntityRelation
+        Possible values found in :class:`~azure.ai.textanalytics.HealthcareEntityRelation`
```

Comment:
I think we got rid of `HealthcareEntityRelation`

Reply:
discussed offline - we got rid of `HealthcareEntityRelationRoleType`